File: CanvasDnD/TRBL.js
/**
 * Created by David on 09/26/16.
 */

class TRBL {
    constructor(){
        this._top = 0;
        this._right = 0;
        this._bottom = 0;
        this._left = 0;
    }

    get top() { return this._top; }
    set top(value) { this._top = value; }

    get right() { return this._right; }
    set right(value) { this._right = value; }

    get bottom() { return this._bottom; }
    set bottom(value) { this._bottom = value; }

    get left() { return this._left; }
    set left(value) { this._left = value; }
}

File: Shared/FormBuilderObjects/Properties/TRBL.cs
namespace Shared.FormBuilderObjects.Properties
{
    public class TRBL
    {
        private double m_top;
        private double m_right;
        private double m_bottom;
        private double m_left;

        public double Top
        {
            get { return m_top; }
            set { m_top = FormBuilder.PxToPt(value); }
        }
        public double Right
        {
            get { return m_right; }
            set { m_right = FormBuilder.PxToPt(value); }
        }
        public double Bottom
        {
            get { return m_bottom; }
            set { m_bottom = FormBuilder.PxToPt(value); }
        }
        public double Left
        {
            get { return m_left; }
            set { m_left = FormBuilder.PxToPt(value); }
        }
    }
}
